daisy/workflow.go

```diff
@@ -153,6 +153,7 @@ type Workflow struct {
 	objects         *objectRegistry
 
 	stepTimeRecords []TimeRecord
+	outputValues    map[string]string
```

Comment:
A more descriptive name would be serialConsoleOutputValues.

Also, since there could be multiple VMs for which we're capturing serial console output (and could thus return serial console output values), information about VM is lost here. We might not need the info about which step/VM returned this value, but since Daisy is a generic framework, some other client might need this info.